pkg/controller/secret/secret_controller.go

```diff
@@ -26,6 +27,8 @@ import (
 )
 
 var log = logf.Log.WithName("secret_controller")
+var reqLogger logr.Logger
```

Comment:
Yeah this isn't right.. you're just instantiating a new log without actually specifying what you want to attach in.

Inside the reconcile loop, something like this would work:
```
reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```